src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, Machine value);
```

Comment:
The `fill` method uses `List<? super Machine>` for the `machines` parameter. This is correct for adding elements, but the comment suggests replacing `Object` with a parameterized value. Ensure that the implementation uses the correct type parameter for `value` to maintain type safety.